src/index.css

```diff
@@ -0,0 +1,70 @@
+:root {
```

Comment:
more vite boilerplate that can be removed?

Reply:
Removed